recipes/physx/4.x.x/conanfile.py

```diff
@@ -363,26 +320,3 @@ def package_info(self):
         self.cpp_info.components["physxextensions"].set_property("cmake_target_name", "PhysX::PhysXExtensions")
         self.cpp_info.components["physxextensions"].libs = ["PhysXExtensions"]
         self.cpp_info.components["physxextensions"].requires = ["physxfoundation", "physxpvdsdk", "physxmain", "physxcommon"]
-
-        # TODO: remove in conan v2 once cmake_find_package* removed
```

Comment:
Don't remove please. CCI recipes have to support both new & legacy generatoes.

Reply:
I added those back, I thought it was ok to remove them since we're moving towards V2, but makes total sense to keep them for backward compatibility. Thanks for the suggestion 🙂